src/Shellbar/__snapshots__/Shellbar.test.js.snap

```diff
@@ -42,9 +42,12 @@ exports[`<Shellbar /> create shellbar 1`] = `
               className="fd-popover__control"
             >
               <span
+                aria-haspopup={true}
                 className="fd-identifier fd-identifier--xs fd-identifier--circle fd-has-background-color-accent-8"
                 onClick={[Function]}
+                onKeyPress={[Function]}
                 role=""
```

Comment:
This `span` should have `role=button`, but that's an issue with the `Identifier` component used in the `ShellBar`. It resets the role. Hence VoiceOver on Chrome doesn't announce the `Identifier` as a pop-up button.